vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -199,45 +200,49 @@ public Retrofit getRetrofit() {
         return mRetrofit;
     }
 
+    /**
+     * @return the current VimeoAccount, or null if it hasn't been set yet
+     */
+    @Nullable
     public VimeoAccount getVimeoAccount() {
-        if (mVimeoAccount == null) {
-            throw new AssertionError("Account should never be null");
-        }
-
         return mVimeoAccount;
```

Comment:
I don't believe this is dangerous to return the reference here, but we could clone it to be safe.